lib/gpprmsya.gd

```diff
@@ -159,6 +159,25 @@ DeclareAttribute("OrbitStabilizingParentGroup",IsPermGroup);
 
 DeclareGlobalFunction("NormalizerParentSA");
 
+#############################################################################
+##
+#F NormalizerSimpleGroupInParentSymmetricViaAutomorphisms( <grp>, [,<aut>]) )
```

Comment:
Now that I think about it, the name is already quite long so we could also call it
`NormalizerSimpleGroupInParentSymmetricGroupViaAutomorphisms`.

Reply:
@fingolfin said in the slack channel that we could just call it `NormalizerOfSimpleGroupInSymmetricGroup`, which I would support. Is it clear enough that this refers to the normaliser on the `MovedPoints`?

Reply:
Also, I don't know what the additional information `ViaAutomorphisms` might help the user? Why do we have it in the function name?